rearrange/environment.py

```diff
@@ -202,6 +203,14 @@ def __init__(
         self.force_cache_reset = force_cache_reset
         self.mode = mode
         self._controller_kwargs = {} if controller_kwargs is None else controller_kwargs
+        self._enhanced_physics_determinism = enhanced_physics_determinism
+
+        self.physics_step_kwargs = {}
+        if self._enhanced_physics_determinism:
+            self.physics_step_kwargs = {
+                "actionSimulationSeconds": 0.26,
+                "fixedDeltaTime": 0.02,
```

Comment:
Presumably these are empirically set, or is there any intuition on these constants here?

Reply:
Yes set empirically. A `fixedDeltaTime` of 0.02 is the Unity default that has high enough resolution to produce reasonable physical trajectories in most instances.